app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name="com.globant.calculatoronboardingkotlin.MainActivity">
```

Comment:
this shouldn't be necessary.. check it pls

Reply:
done, thanks